### ISSUE TITLE
Add responsiveness

### ISSUE THREAD

**Issue by michaella23** [jeffreycharters/station#48]
Write media queries for various elements to change the layout at different viewport widths. 🤠

**Comment by michaella23** [comment]
@Atulsoam562003 yes, feel free! This may be an ongoing issue, but if you see any features right now that need help go ahead!

**Comment by Atulsoam562003** [comment]
hey, please review my commits.

**Comment by michaella23** [comment]
Hero section needs to be retooled for smaller screens. I think a grid display is being used currently.

**Comment by AdityaShaw1** [comment]
Hlo pls assign me to this issue

**Comment by michaella23** [comment]
Hi @AdityaShaw1 ! I realize this issue is a little vague. Would you mind taking a look at the code base and let me know a few specific changes you would like to make? You can comment here or open a new issue, and I'll assign it to you. Thanks! 🙂
